docs: add some credentials to sign in with fake users (#62)
comarev/comarev-dashboard: README.md

```
yarn start

```
After application boot, you will see the login screen

> and you can is sign in with one of these following accounts:

Reviewer: ```suggestion
and you can sign in using one of the credentials below:
```
